### ISSUE TITLE
[CLOSED] Clarify the status of shared_optimizer = 0

### ISSUE THREAD

**Issue by jeswan** [nyu-mll/jiant-v1-legacy#597]
<a href="https://github.com/sleepinyourhat"><img src="https://avatars1.githubusercontent.com/u/1284441?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [sleepinyourhat](https://github.com/sleepinyourhat)**
_Friday Apr 26, 2019 at 22:13 GMT_
_Originally opened as https://github.com/nyu-mll/jiant/issues/597_

----

From the readme: "We have partial support for per-task optimizers (shared_optimizer = 0), but checkpointing may not behave correctly in this configuration. In the per-task case, we stop training on a task when its patience has run out or its optimizer hits the minimum learning rate."

If there's a reason people might want to use this, clean it up and explain it a bit more clearly. If not, let's just remove this option.
